openmls/src/group/core_group/create_commit.rs

```diff
@@ -121,18 +133,22 @@ impl CoreGroup {
         }
 
         // Apply proposals to tree
-        let apply_proposals_values =
-            self.apply_proposals(&mut diff, backend, &proposal_queue, &[])?;
+        let apply_proposals_values = self
+            .apply_proposals(&mut diff, backend, &proposal_queue, &[])
+            .map_err(|_| CreateCommitError::OwnKeyNotFound)?;
```

Comment:
Is this a legitimate case that can happen or is this a library error where there's a bug in OpenMLS? I think this should either be a library error or get more context in the error description what happened.